pkg/apis/pipeline/v1alpha1/build_gcs_resource.go

```diff
@@ -29,25 +29,34 @@ import (
 var (
 	buildGCSFetcherImage = flag.String("build-gcs-fetcher-image", "gcr.io/cloud-builders/gcs-fetcher:latest",
 		"The container image containing our GCS fetcher binary.")
-	buildGCSUploaderImage = flag.String("build-gcs-uploader-image", "gcr.io/cloud-builders/gcs-uploader:latest",
-		"The container image containing our GCS uploader binary.")
 )
 
 // GCSArtifactType defines a type of GCS resource.
 type GCSArtifactType string
 
 const (
 	// GCSArchive indicates that resource should be fetched from a typical archive file.
+	//
+	// Deprecated: Use GCSZipArchive instead.
 	GCSArchive GCSArtifactType = "Archive"
```

Comment:
can we make a follow up issue (unless you did already!) to remove this?

Reply:
Do you have any concerns about me just removing it in this PR?

Reply:
if we're sure no one is using it, im okay with it but we'll need a few more owners to sign off on it (2 more so we have a total of 4? 1/2 + 1)